code/game/machinery/iv_drip.dm

```diff
@@ -494,6 +496,9 @@
 	return attack_hand(user)
 
 /obj/machinery/iv_drip/attack_hand(mob/user as mob)
+	if(user.incapacitated())
+		to_chat(usr, SPAN_WARNING("You are in no shape to do this."))
```

Comment:
```suggestion
		to_chat(user, SPAN_WARNING("You are in no shape to do this."))
```

Reply:
Done! Most (if not all) improper ``usr`` should be McGone now